src/App.jsx

```diff
@@ -14,6 +15,10 @@ function App() {
         <Route path="/" element={<Home />}></Route>
         <Route path="/articles" element={<ArticlesList />}></Route>
         <Route path="/articles/:article_id" element={<Article />}></Route>
+        <Route
```

Comment:
As before, this route is not needed if you're rendering comments in the article component.